cmd/auth-vue/src/pages/SelectProvider.vue

```diff
@@ -0,0 +1,48 @@
+<!--
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+    <div class="container mx-auto flex  flex-wrap pt-4 pb-12">
+        <div class="w-full md:w-1/3 p-6 flex flex-col flex-grow flex-shrink">
+            <div class="flex-none mt-auto bg-white  rounded-b rounded-t-none overflow-hidden p-6">
+                <div class="flex items-center justify-center">
+                    <div class="bg-white shadow-lg border rounded text-center text-black px-32 pt-8 pb-8 md:flex-wrap md:justify-between">
+                        <p class="text-xl">Select Sign-In Partner</p>
+                        <div class="flex items-center  justify-center">
+                            <a href="/auth/google/login" class="mx-auto bg-gray-100 border lg:mx-0 hover:underline my-4 py-2 px-8 shadow-lg">
+                                <img class="object-scale-down h-8 w-32" alt="Google &quot;G&quot; Logo"
+                                     src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Google_%22G%22_Logo.svg/512px-Google_%22G%22_Logo.svg.png"/>
+                                Google
+                            </a>
+                            <a href="" class="mx-auto bg-gray-100 lg:mx-0 border hover:underline my-4 py-2 px-8 shadow-lg">
+                                <img class="object-scale-down h-8 w-32"  alt="Google &quot;G&quot; Logo"
+                                     src="https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg"/>
```

Comment:
Just a thought... Would it be a good idea if these resources are hosted internally? Just in case the external sitre goes down?

Reply:
@talwinder50 WDYT?

Reply:
I dont have very strong preference , Linking and embedding both have their pros and cons. Embedded images on git has to be stored on git lfs for the efficient storage purpose. If its not bothersome, for now i can set up git-lfs and add these images to the that in the following pr.

Reply:
I would probably vote for hosting it internally - I'm not sure if wikimedia ever moves their stuff around... but I think it would be fair to do it in a separate PR